tests/unit/cli/statistics_extractor.py

```diff
@@ -0,0 +1,59 @@
+import unittest
+from pathlib import Path
+
+from rfhub2.cli.statistics_extractor import StatisticsExtractor
+
+FIXTURE_PATH = Path.cwd() / "tests" / "fixtures" / "statistics"
+SUBDIR = FIXTURE_PATH / "subdir"
+SMALL_OUTPUT_XML = SUBDIR / "output.xml"
+
+KEYWORD_1 = {"library": "BuiltIn", "name": "Log", "elapsed": 1}
+KEYWORD_2 = {"library": "BuiltIn", "name": "Comment", "elapsed": 1}
+KEYWORD_3 = {"library": "BuiltIn", "name": "Should Be True", "elapsed": 1}
+KEYWORD = [KEYWORD_1, KEYWORD_2, KEYWORD_3, KEYWORD_1, KEYWORD_2, KEYWORD_3]
+
+STATISTICS_1 = {
+    "collection": "BuiltIn",
+    "keyword": "Log",
+    "execution_time": "2019-12-25 11:38:08.868000",
+    "times_used": 2,
+    "total_elapsed": 2,
+    "min_elapsed": 1,
+    "max_elapsed": 1,
+}
+STATISTICS_2 = {

```

Comment:
you could reuse first statistics dict fields in the next dicts to make it more concise: STATISTICS_2 = {
        **STATISTICS_1,
        "keyword": "Comment"
    }

Reply:
Done

Reply:
it looks like it is still here

Reply:
this time it's ok :)